package.json

```diff
@@ -29,6 +29,7 @@
     "start": "yarn build && yarn serve",
     "build": "yarn build-scss && yarn build-js",
     "build-scss": "yarn run build-js && sass --load-path=node_modules --embed-sources --style=compressed scss:build/css && postcss --use autoprefixer --replace 'build/css/**/*.css' --map",
+    "build:essential": "yarn run build-js && sass --load-path=node_modules --embed-sources --style=compressed scss/build.scss:build/css/build.css scss/docs:build/css/docs && postcss --use autoprefixer --replace 'build/css/**/*.css' --map",
```

Comment:
Not sure why it's here. It was merged separately.
Could you rebase on top of fresh `main` branch?